### ISSUE TITLE
Improve macOS installer

### ISSUE THREAD

**Issue by lassoan** [Slicer/Slicer#5356]
## Is your feature request related to a problem? Please describe.

Users are often unsure how to properly install Slicer on Mac. Installation instructions are available on the web (https://slicer.readthedocs.io/en/latest/user_guide/getting_started.html#installing-3d-slicer) but we cannot expect users to look up instructions.

## Describe the solution you'd like

Improve the dmg file by adding these into the main folder:
- a README in the mac dmg folder that explains how to run the unsigned builds by right clicking and selecting Open twice (denied the first time and then works), and a link to Slicer documentation and Slicer forum
- an arrow to indicate to the user to drag Slicer.app into the Applications folder

**Comment by fepegar** [comment]
The easiest option for Terminal users is running `brew install --cask slicer`. I think it would be useful to add this to the docs.

**Comment by lassoan** [comment]
Good idea @fepegar. Which Slicer version does this command install? Is there an option to install stable or preview? Can you provide commands for update and uninstall? Would youind sending a pull request with the suggested changes? Thank you.

**Comment by fepegar** [comment]
First, users need to install the [Homebrew](https://brew.sh/) package manager (which is great). Homebrew casks can be used to install applications without any downloading, extracting, clicking, dragging, etc. To install the stable version:

```shell
brew install --cask slicer  # install
brew upgrade slicer  # upgrade
brew uninstall slicer  # uninstall
```

as specified in the [`slicer`](https://formulae.brew.sh/cask/slicer) cask website. Currently, this cask is being [manually updated](https://github.com/Homebrew/homebrew-cask/commits/3eabc3b6ff784c80d07d3ed9b6d6771d9e64f482/Casks/slicer.rb). There has been some [discussion](https://discourse.slicer.org/t/slicer-installation-on-mac-using-homebrew/933/24?u=fernando) about updating casks automatically, but that is not happening.

The preview version can be installed using [`homebrew-cask-versions`](https://github.com/Homebrew/homebrew-cask-versions#usage):

```shell
brew tap homebrew/cask-versions  # only once
brew install --cask slicer-preview  # install
brew upgrade slicer-preview  # upgrade
brew uninstall slicer-preview  # uninstall
```

That cask doesn't need to be updated, it will always install the latest preview. This last cask was once [automatically purged](https://github.com/Homebrew/homebrew-cask-versions/commit/5066e55bce491176a370e9f10bee17b240afd712) because of the lacks of downloads. I guess that won't happen if this documented better.

More context:
- [HomebrewCask Lab](https://www.slicer.org/wiki/Documentation/Labs/HomebrewCask) on Slicer wiki
- [Slicer installation on Mac using homebrew](https://discourse.slicer.org/t/slicer-installation-on-mac-using-homebrew/933)
- [Add Slicer Nightly to Homebrew (macOS)](https://discourse.slicer.org/t/add-slicer-nightly-to-homebrew-macos/811)

Should I add all this to the PR?

**Comment by lassoan** [comment]
> Should I add all this to the PR?

Yes, please! Split it two two parts: instruction for users should go to the user's manual - install section. All information that describes how this works internally, how to maintain, etc. Should go to the developer section. You could move contents of the lab page so that everything is in one place.
